addOns/bruteforce/src/main/java/org/zaproxy/zap/extension/bruteforce/BruteForce.java

```diff
@@ -352,4 +353,12 @@ public void setOnlyUnderDirectory(boolean onlyUnderDirectory) {
             this.recursive = true;
         }
     }
+
+    public void setScanId(int scanId) {
```

Comment:
This could have just package visibility so external code doesn't accidentally change it.